### ISSUE TITLE
Tailing semicolon cause ParseException

### ISSUE THREAD

**Issue by xkommando** [javaparser/javaparser#134]
While parsing Android 19 source, I encountered ParseException caused by double semicolons in "import" statements.
The class is `com.android.mediadump.VideoDumpActivity` , and it can be found here https://android.googlesource.com/platform/frameworks/base/+/cd92588/media/tests/MediaDump/src/com/android/mediadump/VideoDumpActivity.java
the problem is caused by the double semicolons at line 22:

```
import android.content.Context;
import android.content.DialogInterface;; // line 22, two semicolons here
import android.content.Intent;
import android.content.SharedPreferences;
```

and the exception is:

```
Exception in thread "main" com.github.javaparser.ParseException: Encountered " "import" "import "" at line 23, column 1.
Was expecting one of:
    <EOF> 
    "abstract" ...
    "class" ...
    "enum" ...
...
```

Regards.

**Comment by ftomassetti** [comment]
Thank you for reporting this. I think we should:
1. find out if the language specifications consider valid to use an orphan semicolon outside methods. After all it is an empty statement which probably does not make sense there
2. probably, whatever it is legit code or not  we should support it anyway

**Comment by ftomassetti** [comment]
It took a while but it should have been fixed by #135 so I am going to close this one, please do not hesitate to reopen if the problem is still present
